src/main/antora/modules/ROOT/pages/repositories/core-concepts.adoc

```diff
@@ -4,6 +4,18 @@
 The central interface in the Spring Data repository abstraction is `Repository`.
 It takes the domain class to manage as well as the identifier type of the domain class as type arguments.
 This interface acts primarily as a marker interface to capture the types to work with and to help you to discover interfaces that extend this one.
+
+[TIP]
+====
+Spring Data considers domain types to be entities, more specifically aggregates.
+So you will see the term "entity" used throughout the documentation that can be interchanged with the term "domain type" or "aggregate".
```

Comment:
@odrotbohm any opinion on wording? We want to make the point that identifiers need sometimes a different handling than properties, yet in both cases we need to express access to these via some name.